src/main/java/sirius/db/mixing/schema/SchemaTool.java

```diff
@@ -165,9 +162,7 @@ private void fillColumns(Connection c, Table table) throws SQLException {
         List<SchemaUpdateAction> result = Lists.newArrayList();
         List<Table> currentSchema = getSchema(db);
 
-        List<Table> sortedTarget = sort(new ArrayList<>(targetSchema));
-
-        syncRequiredTables(result, currentSchema, sortedTarget);
+        syncRequiredTables(result, currentSchema, targetSchema);
```

Comment:
dritter parameter heißt in der deklaration "sortedTarget", die Liste ist nicht mehr sortiert oder? eventl. sollte man das umbenennen